src/CoreBundle/private/core/TModelBase.class.php

```diff
@@ -697,6 +697,9 @@ final public function __invoke(Request $request, $isLegacyModule)
             $sViewName = $this->aModuleConfig['view'];
             $viewMapperConfig = $oModule->getViewMapperConfig();
             $mappers = $viewMapperConfig->getMappersForConfig($sViewName);
+            if (null === $mappers) {
+                throw new MapperException(sprintf('No mappers found/config not found for %s for view %s', $this->aModuleConfig['model'], $sViewName));
+            }
```

Comment:
A return of `null` indicates that the whole config is missing. It is possible for a config to have no mappers and this would not be an error condition.
Therefore in addition we should check `$sViewPathReference` in line 708 for null as this would indicate the same thing - being there isn't a config for this view name.

Reply:
I moved the check to the view path (and that up).

Reply:
ah yes, this is better. But I would still throw an exception if the the mappers are null, because this would indicate an error that can happen after restructuring the code. Currently it is a manual invariant and might be ok, but in the future it might not be.

Reply:
I wouldn't say that null is different from an empty array here.

Both signify, there "isn't anything".

Both simply do not iterate now.

Reply:
Yes, but the implementation uses `null` as a sentinal value for "there is no such config". No mappers would be indicated by an empty array. Yes, this is bad API design, but what can you do. ;)

Reply:
Don't know the implementation from the outside...

Reply:
Inserted the same check again.